expo-maps/src/Maps.types.ts

```diff
@@ -71,30 +93,23 @@ export type PolygonProps = PropsWithChildren<{
   points: Point[];
 }>;
 
-export type MarkerProps = PropsWithChildren<
-  {
-    title?: string;
-    snippet?: string;
-    icon?: string;
-    defaultMarkerColor?: number | MarkerColor;
-    draggable?: boolean;
-    anchorU?: number;
-    anchorV?: number;
-    opacity?: number;
-  } & Point
->;
-
-export type MarkerObject = {
-  type: 'marker';
+export type MarkerOptions = {
   title?: string;
   snippet?: string;
   icon?: string;
-  defaultMarkerColor: number;
-  draggable: boolean;
+  defaultMarkerColor?: number | MarkerColor;
+  draggable?: boolean;
```

Comment:
No need to make them capable of having undefined values since in Maps.tsx we set default values if user does not provide them